src/app/services/search.service.ts

```diff
@@ -116,6 +127,10 @@ export class ResearchStudySearchEntry {
     }
     return '';
   }
+  get matchLikelihood(): string | null {
+    const matchValue = {0: "No Match", 0.5: "Possible Match", 1: "Likely Match"};
+    return this.search ? matchValue[this.search.score] : null;
+  }
```

Comment:
This won't work for values that don't exactly match the search value. A match value of 0.2 will return `null`.

Reply:
That's true, but unless we want to show the score as a rating out of 1, we need to do some mapping. For now, I added a note about this mapping in the service template, but we can also just use numbers out of 1 if we think that's preferable.

Reply:
Switched mapping to range